### PR TITLE
The mysql version check inverted

### PR DESCRIPTION
**Pull request by waynegemmell**
The mysql version check that was supposed to test whether the version is greater than or equal to  8.0.11 was testing for less than or equal to 8.0.11. This then used the password(_) function in the incorrect context.

### Tests written?

No

### Commits signed with GPG?

No

### CODE REVIEW

**Comment by waynegemmell** [comment]
This was supposed to go in against tag 2018.3.3. I'm not sure what went wrong. Selecting the tag on the pull request screen sent me into a code comparison mode.

File: salt/modules/mysql.py [hunk 1]
```diff
@@ -1242,7 +1242,7 @@ def user_exists(user,
         else:
             qry += ' AND ' + password_column + ' = \'\''
     elif password:
-        if salt.utils.versions.version_cmp(server_version, '8.0.11') <= 0:
+        if salt.utils.versions.version_cmp(server_version, '8.0.11') >= 0:
```
Comment (rares-pop) [inline salt/modules/mysql.py]: what about salt.utils.versions.LooseVersion(server_version) >= '8.0.11'

**Comment by garethgreenaway** [comment]
@waynegemmell Looks like your local branch was against `develop`, switching the base branch in the PR results in a lot of changes beyond your change.  That's okay though, we can merge this into `develop` then back port it to 2018.3.

**Comment by garethgreenaway** [comment]
Looking closer at the code already committed, this was fixed in https://github.com/saltstack/salt/pull/49918.
Closing this PR out.

**Pull request closed without merging**